config/deploy.rb

```diff
@@ -48,3 +48,14 @@
     end
   end
 end
+
+namespace :memcached do
+  desc "Flushes memcached instance cache"
+  task :flush do
+    on roles(:app) do
```

Comment:
You can select just one using primary() I think
http://capistranorb.com/documentation/advanced-features/property-filtering/